esp/platform/espp.cpp

```diff
@@ -426,6 +427,11 @@ int init_main(int argc, const char* argv[])
     //save off generated config to register with container.  Legacy can always reference the config file, application based ESP needs generated config saved off
     Owned<IPropertyTree> appConfig;
 
+    TraceManager traceManager("esp");
+    auto espTracer = traceManager.getTracer();//All instances of this esp process can share the same tracer same as TraceManager::getTracer("esp")
+    auto rootEspProcspan = espTracer->StartSpan(__func__); //Dummy top level span for the esp process
```

Comment:
Is this dummy span really needed?  If so, at least for now, I would completely hide it from a process that handles requests.  Even for job type processes, like say eclagent, I think.

Reply:
Not required, the component owner will define the appropriate spans, this illustrates a top level span for the current function